lightning/src/chain/channelmonitor.rs

```diff
@@ -160,119 +164,138 @@ pub enum MonitorEvent {
 	CommitmentTxBroadcasted(OutPoint),
 }
 
-/// Simple structure send back by ManyChannelMonitor in case of HTLC detected onchain from a
+/// Simple structure send back by `chain::Watch` in case of HTLC detected onchain from a
```

Comment:
nit: update comment: "Simple structure *sent* back by `chain::Watch` when an HTLC is detected onchain from a forwarding channel. This info is needed to update the corresponding HTLC in the originating channel."